docs/dg/projectManagement.md

```diff
@@ -21,6 +21,27 @@ This page contains information about project management tasks. The target audien
 
 <!-- ==================================================================================================== -->
 
+## Making a release on Github
+
+Before making a release, please check the following prerequisite.
+
+* Ensure that you have a proper version of **JDK `1.8.0`** installed (==Not **JDK `11`** or other later version==).
+* Ensure that the `JAVA_HOME` environment variable is correctly set to your JDK installation directory. You can refer to the [JDK Installation Guide](https://docs.oracle.com/cd/E19182-01/821-0917/inst_jdk_javahome_t/index.html).
+
+To make a release for RepoSense on Github, please follow the `Creating a release` section in the [Github Docs](https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository).<br>
+  
+Take note of the following the section above:
```

Comment:
Maybe can reword this? The sentence is a little weird.

Reply:
Sure, no problem